lib/hashie/extensions/dash/indifferent_access.rb

```diff
@@ -24,8 +24,11 @@ module ClassMethods
           # defined.
           def property?(name)
             name = translations[name.to_sym] if translation_for?(name)
-            name = name.to_s
-            !!properties.find { |property| property.to_s == name }
+            if name.is_a? ::Array
+              name.all? { |att| !!properties.find { |property| property.to_s == att.to_s } }
```

Comment:
Does name require a translation again in the `Array` case? Should come up with a nested test case with translations at all levels.

If so, then `name.all? { |att | property?(att) }`?